src/contracts/protocol/libraries/logic/ReserveLogic.sol

```diff
@@ -36,6 +39,8 @@ library ReserveLogic {
     uint256 variableBorrowIndex
   );
 
+  event ReserveUsedAsCollateralDisabled(address indexed reserve, address indexed user);
```

Comment:
can this be removed?

Reply:
good catch, was a leftover from when the `eliminateReserveDeficit` was part of ReserveLogic. Will create a pr to bundle your feedback & bump revisions.